src/scripts/browserDeviceProfile.js

```diff
@@ -707,33 +707,29 @@ define(['browser'], function (browser) {
             }
         }
 
-        profile.CodecProfiles.push({
-            Type: 'Video',
-            Codec: 'h264',
-            Conditions: [
-                {
-                    Condition: 'NotEquals',
-                    Property: 'IsAnamorphic',
-                    Value: 'true',
-                    IsRequired: false
-                },
-                {
-                    Condition: 'EqualsAny',
-                    Property: 'VideoProfile',
-                    Value: h264Profiles,
-                    IsRequired: false
-                },
-                {
-                    Condition: 'LessThanEqual',
-                    Property: 'VideoLevel',
-                    Value: maxH264Level.toString(),
-                    IsRequired: false
-                }
-            ]
-        });
+        const h264CodecProfileConditions = [
```

Comment:
Is `const` safe to use in 10.6? @MrTimscampi

Reply:
Yep, Babel converts everything back to ES5.